mmdeploy/pytorch/functions/interpolate.py

```diff
@@ -27,9 +27,8 @@ def interpolate__ncnn(ctx,
 
     input_size = input.shape
     if scale_factor is None:
-        scale_factor = [
-            s_out / s_in for s_out, s_in in zip(size, input_size[2:])
-        ]
+        scale_factor = [(s_out / s_in + 0.00001)
```

Comment:
why do we add this small value, this rewriting would affect all models.

Reply:
Sorry，the last commit committed the wrong version，just add this small value the scale_factor are not integers. Therefore, there may be a rounding error. For example, if the size is 6x6, the resize is 32x32, and the scale_factor is 5.333333333... , the number of bits that can be retained during the conversion is limited, so the size of resize becomes 31.999999 when computed with scale_factor, which is rounded down to 31x31.

Reply:
@lansfair Hi, could we rewrite `resize` and `Upsample` for in mmseg [code](https://github.com/open-mmlab/mmsegmentation/blob/master/mmseg/ops/wrappers.py) to use `size` instead of `scale factor` for ncnn. It seems resize input is `size` for instance in [here](https://github.com/open-mmlab/mmsegmentation/blob/a68e7c2823f6c9c79eb2fea68ecbf626aebd2b85/mmseg/models/segmentors/encoder_decoder.py#L76)